privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ToggleReportsDataStore.kt

```diff
@@ -155,8 +155,8 @@ class SharedPreferencesToggleReportsDataStore @Inject constructor(
 
     override suspend fun getPromptsDismissed(): List<String> =
         withContext(dispatcherProvider.io()) {
-            return@withContext store.data.first().let { prefs ->
-                prefs[TOGGLE_REPORTS_PROMPTS_DISMISSED]?.toList() ?: emptyList()
+            return@withContext store.data.firstOrNull().let { prefs ->
```

Comment:
`store.data` never returns an empty flow, so using `firstOrNull()` is not necessary. Docs recommend using `first()` for accessing store snapshot ->
https://developer.android.com/reference/kotlin/androidx/datastore/core/DataStore#data()